### ISSUE TITLE
stabilityai/stable-diffusion-2-base is missing fp16 files

### ISSUE THREAD

**Issue by ranjansingh11** [huggingface/diffusers#5492]
Hi Team,

My team is trying to fine-tune a sample and we are using model **stabilityai/stable-diffusion-2-base** . Due to this we are stuck at the **Training setting** phase.

Command we are using 

!accelerate launch train_dreambooth.py \
  --pretrained_model_name_or_path=$MODEL_NAME \
  --pretrained_vae_name_or_path="stabilityai/sd-vae-ft-mse" \
  --output_dir=$OUTPUT_DIR \
  --revision="fp16" \
  --with_prior_preservation --prior_loss_weight=1.0 \
  --seed=1337 \
  --resolution=256 \
  --train_batch_size=1 \
  --train_text_encoder \
  --mixed_precision="fp16" \
  --use_8bit_adam \
  --gradient_accumulation_steps=1 \
  --gradient_checkpointing \
  --learning_rate=3e-6  \
  --lr_scheduler="constant" \
  --lr_warmup_steps=0 \
  --num_class_images=25 \
  --sample_batch_size=4 \
  --max_train_steps=6000 \
  --save_interval=1000 \
  --save_sample_prompt="a very cool superhero dog" \
  --concepts_list="concepts_list.json"
  
  
  
  **Error**
  
 **The Diffusers team and community would be very grateful if you could open an issue: https://github.com/huggingface/diffusers/issues/new with the title 'stabilityai/stable-diffusion-2-base is missing fp16 files' so that the correct variant file can be added.**
 
 Can the team look into this?

**Comment by patrickvonplaten** [comment]
Please don't use `--revision="fp16"`. Instead it would be great if you could use `variant="fp16"` here: https://github.com/huggingface/diffusers/blob/48ce118d1ccbd73788f2b2b75c83d9abadeda4bd/examples/dreambooth/train_dreambooth.py#L142 

so that you can download the following weights: https://huggingface.co/stabilityai/stable-diffusion-2-base/blob/main/unet/diffusion_pytorch_model.fp16.safetensors


We should probably add a new `--variant` argument here (cc @sayakpaul )

**Comment by ranjansingh11** [comment]
Sure @patrickvonplaten. Will use **variant="fp16"** and check

**Comment by patrickvonplaten** [comment]
Feel free to open a PR to add it, should be quite a simple change :-)
